WLC/utils/gui.py

```diff
@@ -1,7 +1,9 @@
 import tkinter as tk
 from os.path import dirname
 from tkinter import filedialog
-from ..code_executor.executor import CodeExecutor, DEFAULT_DOCKER_PORT
+
+from ..code_executor.code_executor import CodeExecutor
+from ..code_executor.abstract_executor import DEFAULT_DOCKER_PORT
 
 
 class Gui(tk.Frame):
```

Comment:
Are we still keeping this as a part of the package? Maybe it's time to purge this?

Reply:
I would just keep it for legacy or remove in a different MR.